docker-compose.yml

```diff
@@ -101,7 +101,7 @@ services:
       context: ./
       dockerfile: ./docker/dotnet.dockerfile
     image: datadog-dotnet
-    command: bash -c "/project/docker/with-profiler-logs.bash wait-for-it servicestackredis:6379 -- /project/docker/with-profiler.bash dotnet /project/samples/Samples.ServiceStack.Redis/bin/Release/netcoreapp2.1/publish/Samples.ServiceStack.Redis.dll"
+    command: bash -c "/project/docker/with-profiler-logs.bash wait-for-it servicestackredis:6379 -- /project/docker/with-profiler.bash dotnet /project/samples/Samples.ServiceStack.Redis/bin/${buildConfiguration}/${publishTargetFramework}/publish/Samples.ServiceStack.Redis.dll"
```

Comment:
Thanks for matching this with the `buildConfiguration` used in the `build` step!